Automattic/themes
# Spearhead: Add related posts block styles

The design for a post suggests that at the end of the post, it shows related posts:

![Screen Shot 2020-10-06 at 11 16 46 AM](https://user-images.githubusercontent.com/5375500/95243542-a3068f80-07c5-11eb-9f82-d395c1f4a792.png)

From a templating perspective, this will be a very easy thing to support when block templates / FSE are stable.

In the interim, my proposal for addressing this (#2523) is by adding styles to the Jetpack related posts block. That way we avoid baking this functionality into a template, and can easily add related posts to a page or post as needed via blocks. This PR adds those styles.

Before | After
------- | -------
![Screen Shot 2020-10-06 at 11 14 19 AM](https://user-images.githubusercontent.com/5375500/95243858-14464280-07c6-11eb-9743-22e8cfa18990.png) | ![Screen Shot 2020-10-06 at 11 13 30 AM](https://user-images.githubusercontent.com/5375500/95243871-17413300-07c6-11eb-8f7f-7b9817c818cb.png)

**To test**
- Check this PR out to a site that has Jetpack enabled, e.g. your sandbox
- Edit a page or post and add a related posts block
- Verify that the styles match the design
